content/en/docs/release-oversight/troubleshooting-payload-failures.md

```diff
@@ -0,0 +1,60 @@
+
+- Loki - Grafana logging stack allowing you search logs and narrow time ranges.  This contains intermittent pod log entries that are not present in the must-gather or gather-extra artifacts.  As well as the kubelet journal logs found under gather-extra and others to aid in searching.
+
+
+- KaaS - Representation of the last known state of the cluster.  Can be useful for navigating the cluster via OC or K9S and interrogating namespaces, logs, etc.
+
+
+- PromCIeus - Metrics from the cluster during the test run loaded in Prometheus.
+
+
+- CI Search - https://search.ci.openshift.org/ - If you have found a particular log entry and want to search bugs and other junit runs to find:
+    - How frequently does it occur
+    - Did it start or spike recently
+    - Is there an open bug against it <br><br>
+
+- Sippy - https://sippy.dptools.openshift.org/sippy-ng/ Used to evaluate recent trends for aggregated jobs and individual tests.  You can filter by Release, Job, or Test as well as variants like platform, network, etc.  If you have found a job or test and want to dig deeper to determine how often does it fail, does it fail more ofter for particular variants, has it recently degraded.
+
+
+- Data Studio - https://datastudio.google.com/navigation/reporting The big hammer with big data.  This is our data warehouse.  Like Sippy but at a larger scale we can search, query and dissect job and test results.
+
+- Test Grid - https://testgrid.k8s.io/ (Search for a dashboard/tab: redhat-openshift ...) 
+
+### Artifacts
+The prow job will have a link to the artifacts collected during the run.  Some of the highlights contained in the artifacts are
+
+- The main build.log.txt in the root of the folder
+- Gather-must-gather 
+    - Includes event-filter.html which can be used to search for events that occurred during the run
+    - Includes must-gather.tar that can be downloaded, untarred and searched / viewed locally
+- Gather-extra
+    - Includes nodes containing the kubelet journal logs that can be downloaded and untarred for searching and viewing locally
+    - events.json a collection of all of the events collected during the run
+- Gather-network
+    - Includes network.tar that can be downloaded, untarred and 
+
+### Troubleshooting steps
+- Look for the obvious, did the job fail?  Is there a failure message that can be connected to a recent commit or service?
+- If there are multiple job failures look to see if the same blocking job is failing in each, or identify the one that is most frequently failing
+    - Investigate those aggregated jobs that fail most frequently
+    - Look at the individual runs for the aggregated jobs and identify the most commonly occurring failures
+- If you can't identify a particular test to focus on based on frequency you either have to rely on your gut or ask for help
+- When you narrow in on an individual test:
+    - Consult the repo and see if any recent commits are obvious candidates to impact it
```

Comment:
For a nightly payload, the list of changes since the previous nightly payload can be obtained from these two places:

* the bottom of the page for that payload (e.g., [a recent 4.11 nightly payload](https://amd64.ocp.releases.ci.openshift.org/releasetag/4.11.0-0.nightly-2022-05-18-053037))
* the "PULL REQUESTS" tab on a payload acceptance page